### DISCUSSION TITLE
How to set a variable to the first *.md file in my current directory?

### DISCUSSION THREAD

**Discussion by halloleo** [casey/just#1605] [Q&A] [answered]
I want to set a Just variable `input` to the (alphabetically) first Markdown filename (*.md file) in my current directory.

Can Just do that? (The only way I came up with was executing an external python script to do this, but that feels cumbersome.)

Thanks for pointers!

**Comment by runeimp** [comment, marked as answer]
I don't believe Just has a function for that. So needs to shell-out with backticks to do something like that. This should work for most POSIX systems.

```just
input := `ls *.md | head -1`

first-recipe:
    echo {{input}}
```
